app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
Tìm hiểu và áp dụng i18n cho toàn bọ text dc hiển thị lên view nhé 
(self check các file html.erb luôn gái nha)